docs/user-guide/systemrequirements.md

```diff
@@ -2,40 +2,39 @@
 
 Before installing Zowe&trade;, ensure that your environment meets the prerequisites.
 
-- [z/OS system requirements](#zos-system-requirements-host)
-  - [Zowe Application Framework requirements](#zowe-application-framework-requirements-host)
-  - [Multi-Factor Authentication for Zowe Desktop](#multi-factor-authentication-for-zowe-desktop)
-    - [Using web tokens for SSO on for ZLUX and ZSS](#using-web-tokens-for-sso-on-for-zlux-and-zss)
-    - [Creating a PKCS#11 Token](#creating-a-pkcs11-token)
-    - [Accessing token](#accessing-token)
-    - [Enabling SSO](#enabling-sso)
-- [Zowe CLI requirements](#zowe-cli-requirements)
-    - [Client-side requirements](#client-side-requirements)
-    - [Host-side requirements](#host-side-requirements)
+- [System requirements](#system-requirements)
```

Comment:
This create a Bulleted list with only one top level bullet. Usual standard is to have at least two bullets. Are all of the following bullets sub-bullets to this top one?

Reply:
You're correct, it should not include the name of the page, "System requirements".

It got that way because I use the "Markdown All in One" VSCode plugin to auto-create the TOC, and it simply scanned every heading on the page including the title. oops.

I could manually edit the TOC (it's just a list of links anyway), but instead there's a  tag you can add to omit a heading from the TOC. Then I'll use the plugin to "update TOC" in vscode. 
Looks like this: 
```
 # System requirements <!-- omit in toc --> 
```